src/lua/fiber.c

```diff
@@ -789,6 +791,14 @@ lbox_fiber_set_joinable(struct lua_State *L)
 	return 0;
 }
 
+static int
+lbox_fiber_is_system(struct lua_State *L)
```

Comment:
I think, that the user should at least have a way to check if the fiber cannot be cancelled, as `lbox_fiber_cancel` doesn't throw an error but just ignores cancell request if `FIBER_IS_SYSTEM` is set. 

We can also throw an error in `lbox_fiber_cancel` (it's even mentioned in docs, that this function  can throw an error if it'not permitted to cancel a fiber).  I don't really like that option, as Cartridge's (possibly, not only its) code will be broken with such change.

Reply:
I don't see what a user could possibly do with the knowledge if a particular fiber is system. Let's not extend the public API for now and silently ignore system fibers in `fiber.cancel`. If somebody asks for a way to determine if a particular fiber is system, we'll consider adding this function or a flag to `fiber.info`.